pynxtools/dataconverter/readers/mpes/reader.py

```diff
@@ -269,6 +269,13 @@ def read(
             eln_data_dict,
         ) = handle_h5_and_json_file(file_paths, objects)
 
+        for key in list(config_file_dict):
+            if "*" in key:
+                value = config_file_dict[key]
+                for dim in x_array_loaded.dims:
+                    config_file_dict[key.replace("*", dim)] = value.replace("*", dim)
```

Comment:
What happens if this key is already present?
Is it fine if this is overwritten like it currently is or should it be treated differently? Should an info or warning be logged in this case?

IMO it would be the cleanest that `*` acts as the default if there is no explicit key/value pair. If there is an explicit key value pair this should be taken. In that way you could always overwrite a certain axis from somewhere else if necessary.

Like so:

```
"/ENTRY[entry]/DATA[data]/AXISNAME_indices[@*_indices]": "@data:*.index",
"/ENTRY[entry]/DATA[data]/AXISNAME_indices[@kx_indices]": "5",
```

this ends updated dict
```
"/ENTRY[entry]/DATA[data]/AXISNAME_indices[@ky_indices]": "@data:*ky.index",
...
"/ENTRY[entry]/DATA[data]/AXISNAME_indices[@kx_indices]": "5",
```

instead of 
```
"/ENTRY[entry]/DATA[data]/AXISNAME_indices[@kx_indices]": "@data:*kx.index",
"/ENTRY[entry]/DATA[data]/AXISNAME_indices[@ky_indices]": "@data:*ky.index",
...
```

Reply:
Done. I also added another condition which allows the indices to be renamed in the config. But for this to work, one has to also manually change the @axes attribute to a list containing the new indices instead of "@data:dims" as it is normally.
For example, to rename "energy" to "E", you do: 
```
"/ENTRY[entry]/DATA[data]/@axes": ["kx", "ky", "E", "delay"],
"/ENTRY[entry]/DATA[data]/AXISNAME_indices[@*_indices]": "@data:*.index",
"/ENTRY[entry]/DATA[data]/AXISNAME_indices[@E_indices]": "@data:energy.index"
...
```
and so on for "axisname" field as well.